xapian-applications/omega/omega.cc

```diff
@@ -174,7 +175,11 @@ try {
 
     hits_per_page = 0;
     auto val = cgi_params.find("HITSPERPAGE");
-    if (val != cgi_params.end()) hits_per_page = atol(val->second.c_str());
+    if (val != cgi_params.end()) {
+	if (!parse_unsigned(val->second.c_str(),hits_per_page)) {
```

Comment:
Missing space after comma here (the patch checking script doesn't seem to catch this, but really ought to).

Reply:
thanks i will fix that.